Sage/carbon
# fix(search): fix height, width and background style bugs - FE 2596

### Proposed behaviour
- The height of the page remains the same as the border of the textbox changes. 
- Background of the component is transparent until focus/active state when then textbox background-color is white.
- Width has a prop to either be 100% or a defined pixel width.

![Search-fixed](https://user-images.githubusercontent.com/56251247/80611196-388d3e80-8a32-11ea-8c48-cf419fba0f16.gif)

### Current behaviour
- Focus the textbox, hover over, the height of the page adjusts as the border of the textbox changes. 
- Background is permanently white.
- Width is fixed to 375px.

![Search-broken](https://user-images.githubusercontent.com/56251247/80611458-89049c00-8a32-11ea-9b09-07e44c069049.gif)

### Checklist


<del>- [ ] Theme support

### Additional context
N/A

### Testing instructions

- Open Storybook
- Navigate to `Design Systems` folder
- Navigate to the `Search` component
- Navigate to `Docs` tab
- Clicking on any of the demo components should not alter the height of the demo container
